mozci/console/commands/push.py

```diff
@@ -587,6 +588,17 @@ def handle(self) -> None:
                     f"<comment>Push {push.branch}/{push.rev} was backedout and all of its failures and the ones of its children were marked as intermittent or marked as caused by another push</comment>"
                 )
 
+            for task in push.tasks:
+                if task.classification != "fixed by commit":
+                    continue
+
+                try:
+                    HgRev.create(task.classification_note[:12], branch=push.branch)
```

Comment:
I tried to modify a task classification by putting nonsense and the warning did not appear, it doesn't look like `HgRev.create()` will ever raise a `PushNotFound` exception.

```
task.classification = "fixed by commit"
task.classification_note = "not a revision"
```

Reply:
Fixed in 069e84b1b6fb209971591f8529d921436eac1396.
"not a revision" will actually cause another exception to be thrown, different than PushNotFound. I'll fix that in another commit.

Reply:
Done in d5764e0eef2185366c29f39f38e82935882cb6e5.